src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -358,7 +358,7 @@ public function testCreateProxyWithRealServiceInstantiator()
         $foo1 = $builder->get('foo1');
 
         $this->assertSame($foo1, $builder->get('foo1'), 'The same proxy is retrieved on multiple subsequent calls');
-        $this->assertSame('Bar\FooClass', get_class($foo1));
+        $this->assertInstanceOf('Bar\FooClass', $foo1);
```

Comment:
this is not the same assertion (it now allows subclasses)

Reply:
Oh, thanks :smile: